package-lock.json

```diff
@@ -4124,7 +4124,8 @@
             },
```

Comment:
we sholudn't need this file modification.

Reply:
Yes, I deleted it.